src/pages/basic-management/employee/list/Index.js

```diff
@@ -0,0 +1,473 @@
+}))
+@Form.create()
+class employeeList extends PureComponent {
+    state = {
+        selectedRows: [],
+        expandForm: false,
+    };
+
+    componentDidMount() {
+        const { dispatch } = this.props;
+        dispatch({
+            type: 'employee/fetch',
+            payload: {},
+        });
+        dispatch({
+            type: 'depart/fetch',
+            payload: {}
+        })
+    }
+
+    handleStandardTableChange = (pagination, filtersArg, sorter) => {
+        const { dispatch } = this.props;
+        const { formValues } = this.state;
+
+        const filters = Object.keys(filtersArg).reduce((obj, key) => {
+            const newObj = { ...obj };
+            newObj[key] = getValue(filtersArg[key]);
+            return newObj;
+        }, {});
+
+        const params = {
+            page: pagination.current,
+            pageSize: pagination.pageSize,
+            ...formValues,
+            ...filters,
+        };
+        if (sorter.field) {
+            params.sorter = `${sorter.field}_${sorter.order}`;
+        }
+
+        dispatch({
+            type: 'user/fetch',
+            payload: params,
+        });
+    };
+
+    handleSelectRows = rows => {
+        this.setState({
+            selectedRows: rows,
+        });
+    };
+
+    columns = [
+        {
+            title: '姓名',
+            dataIndex: 'realName',
+        },
+        {
+            title: '工号',
+            dataIndex: 'empNo',
+        },
+        {
+            title: '性别',
+            dataIndex: 'sex',
+        },
+        {
+            title: '所属部门',
+            dataIndex: 'depart.name',
+        },
+        {
+            title: '所属单位',
+            dataIndex: 'departUnit.name',
+        },
+        {
+            title: '在职状态',
+            dataIndex: 'status',
+            filters: [
+                {
+                    text: status[0],
+                    value: 0,
+                },
+                {
+                    text: status[1],
+                    value: 1,
+                },
+            ],
+            render(val) {
+                return <Badge status={statusMap[val]} text={status[val]} />;
+            },
+        },
+        {
+            title: '录入人',
+            dataIndex: 'addBy',
+        },
+        {
+            title: '录入日期',
+            dataIndex: 'addOn',
+        },
+        {
+            title: '修改人',
+            dataIndex: 'editBy'
+        },
+        {
+            title: '修改日期',
+            dataIndex: 'editOn',
+        },
+        {
+            title: '操作',
+            fixed: 'right',
+            render: (text, record) => (
+                <Fragment>
+                    <a onClick={() => this.handleEdit(record)}>编辑</a>
+                    <Divider type="vertical" />
+                    <Popconfirm
+                        title="你确定要删除此记录吗?"
+                        onConfirm={() => this.handleDelete(record)}
+                        okText="确定"
```

Comment:
Codacy found an issue: [Expected indentation of 22 space characters but found 24.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)